sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+- The default service version is now "7.5-preview.1".
```

Comment:
Is this version correct @nisha-bhatia? I think this might be `V7_6_Preview_1`

```suggestion
- The default service version is now "V7_6_Preview_1".
```